text/0009-const-init-shape-castable.md

```diff
@@ -44,9 +46,9 @@ A method `def const(self, obj):` is added on `ShapeCastable`.
 
 The `Signal(shape, reset=)` constructor is changed so that if `isinstance(shape, ShapeCastable)`, then `shape.const(reset)` is used instead of `reset`.
 
-The `.const()` instance method is implemented on `Layout` to accept a `Sequence` or `Mapping` instance and returns a `Const` with a bit pattern that has the fields set to the given values. Overlapping fields are written in the order of iteration of the input. If the field shape is a shape-castable object, then the value for that field is computed using `Const.cast(value, field.shape)`.
+The `.const()` instance method is implemented on `Layout` to accept a `Sequence` or `Mapping` instance and returns a `View` over a `Const` with a bit pattern that has the fields set to the given values. Overlapping fields are written in the order of iteration of the input. If the field shape is a shape-castable object, then the value for that field is computed using `Const.cast(value, field.shape)`.
```

Comment:
There is no such thing as a `View` in general; it is not something that exists outside of `lib.data`, and therefore cannot be a part of this amendment.

Reply:
`Layout` is also part of `lib.data` so I figure this point is `lib.data`-specific.

Reply:
Oh, no, you are absolutely right. Sorry, I'm extremely tired.